CONTRIBUTORS

```diff
@@ -18,13 +20,12 @@ Ben Harshbarger, Cray Inc.
 Tom Hildebrandt, Cray Inc.
 David Iten, Cray Inc.
 Vassily Litvinov, Cray Inc.
-Simon Lund, Københavns Universitet (University of Copenhagen)
 Tom MacDonald, Cray Inc.
-Phil Nelson, Western Washington University
+Damian McGuckin, Pacific ESI
```

Comment:
I'd put Pacific Engineering Systems International
(spelled out) - otherwise it's fine.